test_windows.js

```diff
@@ -0,0 +1,19 @@
+const spawn = require('child_process').spawn;
+var path = require('path');
+var semver = require('semver');
+
+process.env.GCLOUD_PROJECT = 0
+process.env.CLOUD_DEBUG_ASSERTIONS = 1
+spawn(path.join('node_modules', '.bin', 'mocha.cmd'),
```

Comment:
Like in the Unix case, we should make sure an unset value of `GCLOUD_USE_INSPECTOR` is being used here and that the script doesn't leak environment variable state.

Reply:
Done.